config/environments/production.rb

```diff
@@ -69,6 +69,16 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    user_name: ENV['SMTP_USER'],
```

Comment:
Are there values for these settings in lastpass?

Reply:
They are common for all of the rails apps that send email... so sort of if you look at bento instead of the non-existent thing settings. Thing settings will be added as we approach completion before we consider production "production".

Reply:
https://mitlibraries.atlassian.net/browse/TI-67

Reply:
👍